tests/retrieval/helpers.py

```diff
@@ -34,10 +34,15 @@ def _compute_sklearn_metric(
 
 
 def _test_retrieval_against_sklearn(
-    sklearn_metric, torch_metric, size, n_documents, query_without_relevant_docs_options
+    sklearn_metric,
+    torch_metric,
+    size,
+    n_documents,
+    query_without_relevant_docs_options,
```

Comment:
not sure if they are correct
```suggestion
    sklearn_metric: Callable,
    torch_metric: Metric,
    size: int,
    n_documents: int,
    query_without_relevant_docs_options: str,
```